app/assets/stylesheets/application.sass.scss

```diff
@@ -89,39 +89,52 @@ $color_app-pale-blue: #ccdff1;
   }
 }
 
-.app-header__navigation-text {
-  @include nhsuk-font(16);
-  border-bottom: nhsuk-spacing(1) solid transparent;
-  border-top: nhsuk-spacing(1) solid transparent;
-  color: $color_nhsuk-blue;
-  display: block;
-  padding: 12px nhsuk-spacing(3);
-  text-decoration: none;
-
-  @include mq($from: large-desktop) {
-    color: $color_nhsuk-white;
-    line-height: normal;
-  }
+.app-logo {
+  margin-right: 12px;
 
-  .nhsuk-icon__chevron-right {
-    fill: $color_nhsuk-grey-3;
-    position: absolute;
-    right: nhsuk-spacing(1);
-    top: 11px;
+  @include mq($from: tablet) {
+    margin-right: 0;
   }
 }
 
-.app-header__menu-toggle--no-search {
-  @extend .nhsuk-header__menu-toggle;
-  @include mq($until: desktop) {
-    right: 0px;
+.app-header {
+  &__container {
+    border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+  }
+
+  &__service-name {
+    display: inline-block;
+    line-height: 1.25;
+    vertical-align: top;
+
+    @include mq($from: tablet, $until: large-desktop) {
+      max-width: 300px;
+    }
+  }
```

Comment:
Given the space between the logo and service name is added to the service name in `nhsuk-frontend`, perhaps we should keep that rule here, removing left padding on the service name until we reach the tablet breakpoint. This means we can remove the need for adaptions to the logo (`.app-logo`) as well. 

```suggestion
  &__service-name {
    display: inline-block;
    line-height: 1.25;
    vertical-align: top;

    @include mq($until: tablet) {
      padding-left: nhsuk-spacing(3);
    }

    @include mq($from: tablet, $until: large-desktop) {
      max-width: 300px;
    }
  }
```

Reply:
Version 1:

```scss
{
  padding-left: nhsuk-spacing(3);

  @include mq($from: tablet) {
    padding-left: 0;
  }
}
```

Version 2:

```scss
{
  @include mq($until: tablet) {
    padding-left: nhsuk-spacing(3);
  }
}
```

Both do the same thing, but is v1 more "mobile-first?" I guess it doesn't really matter.

Reply:
Ah, I did try this version. Using padding instead of margin on the logo has the downside of not looking good in the "Miniscule" size:

![image](https://github.com/nhsuk/record-childrens-vaccinations/assets/1650875/6095882e-8645-441e-8156-df2db6f0173d)

But after creating that screenshot I realised that this particular viewport probably doesn't exist in practice. So let's simplify to padding.